src/test/scala/uk/gov/hmrc/perftests/ndrc/JourneyRequests.scala

```diff
@@ -27,12 +27,17 @@ import scala.concurrent.duration._
 object JourneyRequests extends HttpConfiguration with ServicesConfiguration with RequestUtils {
 //Representative multi-entry journey
 
-  val navigateToAuthLoginStubPage =
+  val headers: Map[String, String] = Map(
+    """Accept""" -> """text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8""",
+    """Cache-Control""" -> """no-cache"""
+  )
```

Comment:
Where those headers are used? I cannot find a usage in the PR.

Reply:
It's used 53 times in that file..

Reply:
The headers was originally part of `HttpConfiguration` but it is not there in the later version of the library, so I brought it into this service.